src/main/java/racingcar/Application.java

```diff
@@ -0,0 +1,13 @@
+package racingcar;
+
+import racingcar.controller.RacingcarController;
+import racingcar.service.CarService;
+
+public class Application {
+    public static void main(String[] args) {
+        RacingcarController racingcarController = new RacingcarController(new CarService());
+        racingcarController.start();
+        racingcarController.run();
+        racingcarController.finish();
+    }
```

Comment:
애플리케이션 에서는 단순히 실행만을 명령하도록 하면 어떨까요?
애플리케이션은 실행만 담당하고 컨트롤러가 행동을 제어하도록 리팩토링 해보면 좋겠어요 😄

```suggestion
public class Application {
    public static void main(String[] args) {
        RacingcarController racingcarController = new RacingcarController(new CarService());
        racingcarController.run();
    }
```

Reply:
Controller 의 run 메소드의 이름을 play로 변경하고 실행을 담당하는 메소드를 생성하여 애플리케이션에서는 실행만 담당하도록 리팩토링 해보았습니다 !